amgut/db/patches/0032.sql

```diff
@@ -0,0 +1,561 @@
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'France', 74);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'French Guiana', 75);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'French Polynesia', 76);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'French Southern Territories', 77);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Gabon', 78);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Gambia', 79);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Georgia', 80);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Germany', 81);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Ghana', 82);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Gibraltar', 83);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Greece', 84);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Greenland', 85);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Grenada', 86);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guadeloupe', 87);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guam', 88);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guatemala', 89);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guernsey', 90);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guinea', 91);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guinea-bissau', 92);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Guyana', 93);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Haiti', 94);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Heard Island and Mcdonald Islands', 95);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Holy See (Vatican City State)', 96);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Honduras', 97);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Hong Kong', 98);
```

Comment:
It is by ISO 3166 standards, so we need to go by that since it's what Google geolocating returns.